quickstart/pom.xml

```diff
@@ -41,90 +38,110 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>com.datastax.oss.quarkus</groupId>
+        <artifactId>cassandra-quarkus-bom</artifactId>
+        <version>${project.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
     </dependencies>
   </dependencyManagement>
   <dependencies>
     <dependency>
-      <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-resteasy</artifactId>
+      <groupId>com.datastax.oss.quarkus</groupId>
+      <artifactId>cassandra-quarkus-client</artifactId>
     </dependency>
-    <!-- Tinkerpop uses JCL directly; redirect JCL calls to JBoss logging  -->
     <dependency>
-      <groupId>org.jboss.logging</groupId>
```

Comment:
See my other comment; not exclusing jcl-over-slf4j makes this dependency useless.